File: angular-app/src/app/inventory-management/models/step/step-response.ts
export class StepResponse {
  id!: string;
  name!: string;
  description!: string;
  creationDate!: string;
  previousStepId?: string;
  previousStepName?: string;
  nextStepId?: string | null;
  nextInventoryStepName?: string | null;
}

File: dotnet-restapi/InventoryManagementApi/DTOs/Step/StepResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace InventoryManagementApi.DTOs.Step
{
    public class StepResponse
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public DateTime CreationDate { get; set; }
        public string NextInventoryStepName { get; set; }
    }
}
